mlflow/server/js/.eslintrc.js

```diff
@@ -122,10 +123,11 @@ module.exports = {
     'max-depth': [2, 4],
     'max-len': ['error', 100, 2, {
       'ignoreUrls': false,
-      'ignoreComments': false,
+      'ignoreComments': true,
       'ignoreRegExpLiterals': true,
       'ignoreStrings': false,
       'ignoreTemplateLiterals': false,
+      'ignorePattern': "^import\\s.+\\sfrom\\s'.+';",  // ignore import statements
```

Comment:
<img width="816" alt="Screen Shot 2020-04-26 at 17 46 59" src="https://user-images.githubusercontent.com/17039389/80302608-f40d6300-87e5-11ea-849a-500daa311c7e.png">

Reply:
We need to keep a max length of 100. Can we try to find some extra config on the prettier side to work around this?